lib/galaxy/tool_util/loader_directory.py

```diff
@@ -171,7 +171,10 @@ def looks_like_xml(path, regex=TOOL_REGEX):
         return False
 
     with open(path, "r") as f:
-        start_contents = f.read(5 * 1024)
+        try:
```

Comment:
we could also use a bytestring regex instead, but I doubt Galaxy will be able to read the file ...